lib/eco/eco.py

```diff
@@ -833,6 +834,15 @@ def contextMenu(self, pos):
         self.btReparse([])
 
     def showPygameTree(self):
+        try:
```

Comment:
This puts more stuff than necessary in the `try` I think? Could we not do something like:

```python
os.environ[...] = ...
try:
    import pygame
except ImportError:
    ...
    return
self.ui.actionPvShow...
```

Reply:
Fixed. The `self.ui` stuff wasn't needed here so I removed it.